tcms/settings/common.py

```diff
@@ -136,8 +136,6 @@
     "django.contrib.auth.middleware.AuthenticationMiddleware",
     "django.contrib.messages.middleware.MessageMiddleware",
     "simple_history.middleware.HistoryRequestMiddleware",
-    "tcms.core.middleware.CheckSettingsMiddleware",
-    "tcms.core.middleware.CheckUnappliedMigrationsMiddleware",
```

Comment:
Commit log should document that these 2 have been removed b/c they are part of our settings interface which can be manipulated and overridden further.